draft-ietf-teep-protocol.md

```diff
@@ -1186,3 +1186,179 @@ ta-uuid = 16
+      04                  # unsigned(4)
+      43                  # bytes(3)
+         010203           # "\x01\x02\x03"
+   02                     # unsigned(2)
+~~~~
+
+## QueryResponse
+
+### CBOR Diagnostic Notation
+
+~~~~
+/ query-response = /
+[
+    2,          / type : TEEP-TYPE-query-response = 2 (fixed int) /
+    2004318071, / token : 0x77777777 (uint), from TAM's QueryRequest /
+    / options : /
+    {
+        5 : 1,  / selected-cipher-suite = 5(mapkey) :/
+                / TEEP-AES-CCM-16-64-128-HMAC256--256-X25519-EdDSA = 
+                      1 (uint .size 8) /
+        6 : 0,  / selected-version = 6 (mapkey) : 0 (uint .size 4) /
+        8 : [ h'010203', h'040506' ]   
+                / ta-list = 8 (mapkey) : 
+                      [ 0x010203, 0x040506 ] (array of bstr) /
+    }
+]
+~~~~
+
+### CBOR Binary Representation
+
+~~~~
+83                        # array(3)
+   02                     # unsigned(2)
+   1a 77777777            # unsigned(2004318071, 0x77777777)
+   A3                     # map(3)
+      05                  # unsigned(5)
+      01                  # unsigned(1) within .size 8
+      06                  # unsigned(6)
+      00                  # unsigned(0) within .size 4
+      08                  # unsigned(8)
+      82                  # array(2)
+         43               # bytes(3)
+            010203        # "\x01\x02\x03"
+         43               # bytes(3)
+            040506        # "\x04\x05\x06"
+~~~~
+
+## TrustedAppInstall
+
+### CBOR Diagnostic Notation
+
+~~~~
+/ trusted-app-install = /
+[
+    3,          / type : TEEP-TYPE-trusted-app-install = 3 (fixed int) /
+    2004318072,  / token : 0x777777778 (uint), generated by TAM /
+    / options :  /
+    {
+        10 : [ ] / manifest-list = 10 (mapkey) : 
+                       [ ] (array of SUIT_Envelope(any)) /
+                 / empty, example purpose only /
+    }
+]
+~~~~
+
+### CBOR Binary Representation
+
+~~~~
+83                        # array(3)
+   03                     # unsigned(3)
+   1a 77777778            # unsigned(2004318072, 0x77777778)
+   A1                     # map(1)
+      0A                  # unsigned(10)
+      80                  # array(0)
+~~~~
+
+## Success (for TrustedAppInstall)
+
+### CBOR Diagnostic Notation
+
+~~~~
+/ teep-success = /
+[
+    5,          / type : TEEP-TYPE-teep-success = 5 (fixed int) /
+    2004318072, / token : 0x777777778 (uint), from TrustedAppInstall /
+]
+~~~~
+
+### CBOR Binary Representation
+
+~~~~
+83                        # array(3)
+    05                    # unsigned(5)
+    1a 77777778           # unsigned(2004318072, 0x77777778)
+~~~~
+
+
+## Error (for TrustedAppInstall)
+
+### CBOR Diagnostic Notation
+
+~~~~
+/ teep-error = /
+[
+    6,          / type : TEEP-TYPE-teep-error = 6 (fixed int) /
+    2004318072, / token : 0x777777778 (uint), from TrustedAppInstall /
+    ERR_RESOURCE_FULL, / err-code : ERR_RESOURCE_FULL = 11 (uint) /
```

Comment:
I'm now thinking we should put ERR_MANIFEST_PROCESSING_FAILED (17) back in the document (removed in PR #60) and instead delete ERR_RESOURCE_FULL, moving that to a SUIT report.

Reply:
Opened PR #66